src/Tools/PrepareTests/MinimizeUtil.cs

```diff
@@ -100,36 +128,59 @@ void writeHydrateFile()
                 .SelectMany(pair => pair.Value.Select(fp => (Id: pair.Key, FilePath: fp)))
                 .GroupBy(fp => fp.FilePath.RelativeDirectory);
             var builder = new StringBuilder();
-            builder.AppendLine("@echo off");
             var count = 0;
             foreach (var group in grouping)
             {
                 foreach (var tuple in group)
                 {
                     var source = Path.Combine(duplicateDirectoryName, getPeFileName(tuple.Id));
                     var destFileName = Path.Combine(group.Key, Path.GetFileName(tuple.FilePath.FullPath));
-                    builder.AppendLine($@"
-mklink /h {destFileName} {source} > nul
-if %errorlevel% neq 0 (
-    echo %errorlevel%
-    echo Cmd failed: mklink /h {destFileName} {source} > nul
-    exit 1
-)");
+                    builder.AppendLine($@"New-Item -ItemType HardLink -Name {destFileName} -Value {source} -ErrorAction Stop | Out-Null");
 
                     count++;
                     if (count % 1_000 == 0)
                     {
-                        builder.AppendLine($"echo {count:n0} hydrated");
+                        builder.AppendLine($"Write-Host '{count:n0} hydrated'");
                     }
                 }
             }
 
-            File.WriteAllText(Path.Combine(destinationDirectory, "rehydrate.cmd"), builder.ToString());
+            File.WriteAllText(Path.Combine(destinationDirectory, "rehydrate.ps1"), builder.ToString());
         }
     }
 
-    [DllImport("Kernel32.dll", CharSet = CharSet.Unicode)]
-    private static extern bool CreateHardLink(string lpFileName, string lpExistingFileName, IntPtr lpSecurityAttributes);
+    private static void CreateHardLink(string fileName, string existingFileName)
+    {
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+        {
+            var success = CreateHardLink(fileName, existingFileName, IntPtr.Zero);
+            if (false && !success)
```

Comment:
I found that including this block caused a ton of output to get included. Perhaps we should just have the tool delete the 'testPayload' directory at the very beginning to simplify the loop of developing it.

Reply:
fixed.